ludwig/datasets/mixins/image_process.py

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env python
+# coding=utf-8
+# Copyright (c) 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+import struct
+from array import array
+from os import path
+import png
+from ludwig.datasets.mixins.process import IdentityProcessMixin
+
+
+class ImageProcessMixin(IdentityProcessMixin):
```

Comment:
Same as above, this functionality is MNIST-specific, and should be moved into the appropriate class.